### ISSUE TITLE
Volume Is Too Low

### ISSUE THREAD

**Issue by jayadevanraja** [ashutoshgngwr/noice#358]
### Describe the bug
The volume is too low

### To Reproduce
1. Start the app
2. Play the preset sounds.

### Expected behavior
The volume should be loud enough to wake us up, from sleep. I can hardly hear it.

### Others
 - Device: Moto G5
 - Android Version: 8.1

**Comment by ashutoshgngwr** [comment]
Hi @jayadevanraja 

I realise that Wake up timer doesn't use "Alarm volume" when starting the preset. As a result, user's media volume can be low and the preset will play at that volume. Is that your concern?

**Comment by JekRock** [comment]
Agree. The volume of some sound could be increased

**Comment by jayadevanraja** [comment]
@ashutoshgngwr 
I am extremely sorry for the late reply.

I started looking at the preset sounds.

I have set all my volume controls like (1) Media Volume, (2) Alarm Volume (3) Ring Volume (4) the volume control inside the app, to maximum. Still I can hear the sounds only when I put my ear near the phone.

I liked the idea behind this app. But it is unusable, since the sounds are all too small. That is all.
